utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -630,6 +630,25 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case 10:
+                {
+                    testCaseName = "fog";
+
+                    Rpp32f gamma[batchSize];
```

Comment:
gamma has not be renamed to gammaTensor in HOST test suite
Please rename here too